src/brasil/gov/tiles/configure.zcml

```diff
@@ -48,4 +48,11 @@
     />
   <!--Exclusão-->
 
+  <class class=".widgets.textlinessortable.TextLinesSortableWidget">
```

Comment:
Este widget é uma customização do widget do cover para mostrar o subtitulo dos itens do tile lista para serem customizados na aba compose.
Acredito que o lugar ideal para este widget é no pacote **collective.nitf**, ja que é nele que é inserido o campo de subtitulo.

Reply:
@hvelarde @rodfersou 
Acho que esse tile vai demorar mais um pouco, não tinha visto que ele dependia do pacote `plone.app.imagecropping` para cortar as imagens.

Quando adicionei a dependência ocorre o conflito:
Error: The requirement ('plone.namedfile>=3.999') is not allowed by your [versions] constraint (3.0.11)

Reply:
me lembro vagamente dessa dependência... serve para cortar as imagens das miniaturas, certo?